src/lexedata/importer/fromexcel.py

```diff
@@ -150,12 +150,12 @@ def match(x, y):
 
             def match(x, y):
                 return x == y
-
-        return [
-            candidate
-            for candidate, properties in self.cache[object.__table__].items()
-            if all(match(properties[p], object[p]) for p in properties_for_match)
-        ]
+        candidates = []
+        for candidate, properties in self.cache[object.__table__].items():
+            properties_in_object = [p for p in properties_for_match if p in object and p in properties]
```

Comment:
This matches objects missing a property that we require.
If our cell parser was slightly dodgy, <or> (with no phonemic transcription given) would match /and/. That sounds very bug-prone to me.
Did you ever encounter a KeyError ln line 157? Where would that come from, and can we complain about that error earlier? Because if there is a KeyError, something is fundamentally wrong with the dataset to be imported, and we should not sweep that error under the rug.

Reply:
Yes, there were KeyErrors, the test script doesn't run with the old structure. I get: 
```
>   all(match(properties[p], object[p]) for p in properties)]
E   KeyError: 'ID'
..\src\lexedata\importer\fromexcel.py:154: KeyError
```

Reply:
Hm. Okay, I'll make an issue out of this as a note for the future and otherwise accept the pull request if it passes Travis.